source/BoardingPanel.cpp

```diff
@@ -72,6 +72,10 @@ BoardingPanel::BoardingPanel(PlayerInfo &player, const shared_ptr<Ship> &victim)
 	for(const auto &it : victim->Outfits())
 		if(!it.first->Get("unplunderable"))
 			plunder.emplace_back(it.first, it.second);
+
+	// Outfits stored in cargo can always be plundered.
+	for(const auto &it : victim->Cargo().Outfits())
+		plunder.emplace_back(it.first, it.second);	
```

Comment:
Ah, hence the oddity I just noticed tonight when I accidentally re-boarded such a ship:

![weird](https://cloud.githubusercontent.com/assets/23544894/25829110/2edddabe-344c-11e7-80d1-cbaeb58a94ad.JPG)

Does remove it happily, leaving -1 of it equipped. Apologies for not picking up on that before posting the PR.

I'll fix it. : )